scripts/travis-check-docs.bash

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+
+# Hello future developers! You are probably wondering whose stupid idea it was
+# to write this script, you just want to get your tests to pass and your code
+# to merge. An excellent sentiment! However, consider the state of the project
+# as a whole. When this was written, the docs and tests of GWM were out of date
+# Many configuration options were undocumented. GWM did strange things
+# everywhere and we couldn't figure out why. We decided that new features and
+# new PRs should require docs. You will probably be at the lab for a relatively
+# short time, so please be considerate of those who come after you.
+
+
+
+# Travis CI runs some commands which look kind of like this. We rely on the
+# repo having a merge commit as the most recent commit.
+# If Travis changes the way it starts up this script might break. Don't
+# despair! You'll probably only need to change PARENT_1 and PARENT_2 below,
+# everything else should work fine.
+#   $ git clone --depth=50 git://github.com/osuosl/ganeti_webmgr.git osuosl/ganeti_webmgr
+#   $ cd osuosl/ganeti_webmgr
+#   $ git fetch origin +refs/pull/36/merge:
+#   $ git checkout -qf FETCH_HEAD
+
+PREV_COMMIT=`git rev-list --parents HEAD -n 1`
+if [[ `echo $PREV_COMMIT | wc -w`  =~ '3' ]]; then
```

Comment:
You should check for `<=`, not `=~`. You can have more than two parents in a merge commit, though it is rare in our workflow.